pandas/core/series.py

```diff
@@ -2576,10 +2576,136 @@ def _needs_reindex_multi(self, axes, method, level):
         """
         return False
 
-    @Appender(generic._shared_docs['align'] % _shared_doc_kwargs)
     def align(self, other, join='outer', axis=None, level=None, copy=True,
               fill_value=None, method=None, limit=None, fill_axis=0,
               broadcast_axis=None):
+        """
+        Align two Series on their axes with the specified join method for each
```

Comment:
I think the summary is too long you can stop after: join method,

Reply:
Ok